servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -111,7 +137,11 @@ def start_job(self,
                       outputs={},
                       outputs_recursive={},
                       task_count=1,
-                      wait=False):
+                      wait=False,
+                      with_test_token=False):
```

Comment:
Wouldn't `True` be a more reasonable default here? Isn't your test the only one that would want this to be false?